docs/ops/deploy.md

```diff
@@ -36,3 +54,10 @@ pd-addrs="tikv://your-pd-addrs:port"
 ```
 redis-cli -p 7369
 ```
+
+If Multi-tenancy is enabled, use server generated token to auth:
```

Comment:
use (-)hyphen to link 'sever' with 'generated', like server-generated token ...

Reply:
fixed